src/component-library/CoinIcon/CoinIcon.style.tsx

```diff
@@ -1,11 +1,22 @@
 import styled from 'styled-components';
 
+import { Sizes } from '../utils/prop-types';
+
 interface IconWrapperProps {
-  size: 'small' | 'large';
+  $size: Sizes;
 }
 
-export const IconWrapper = styled.div<IconWrapperProps>`
-  width: ${(props) => (props.size === 'small' ? '2.625rem' : '3.755rem')};
+const IconWrapper = styled.div<IconWrapperProps>`
+  width: ${(props) => {
+    switch (props.$size) {
+      case 'small':
+        return '2.625rem';
+      case 'medium':
+        return '3.755rem';
+      case 'large':
+        return '5.625rem';
```

Comment:
nit: Maybe these values could be defined in the theme config file, what's your opinion on that?

Reply:
yes, lets put it in the theme.